x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/constants/parameters_definition.tsx

```diff
@@ -538,23 +544,21 @@ export const PARAMETERS_DEFINITION = {
   },
   dynamic: {
     fieldConfig: {
-      label: i18n.translate('xpack.idxMgmt.mappingsEditor.dynamicFieldLabel', {
-        defaultMessage: 'Dynamic',
-      }),
-      type: FIELD_TYPES.CHECKBOX,
       defaultValue: true,
     },
     schema: t.boolean,
+    documentation: {
```

Comment:
I thought we were moving away from defining the doc links here, although I'm having a hard time tracking down the discussion now 😄. I think it would be nice if we could be consistent for all parameters. /cc @cjcenizal

Reply:
I haven't' seen the discussion, but to the contrary, the idea I had was to have a single source of truth for types and parameters with the doc links, the default values, the copy texts... One place to look at if there are any changes.

As a side note in a future refactor I'd like to remove the tight coupling of the mappings editor with Index management by providing the `esDocsBase` and `kibanaDocsBase` as required context props and put back the documentation service inside the mappings editor. I prefer hexagonal architecture where INs and OUTs are clearly defined (without relative imports outside the mappings editor folder), which would make the mappings editor truly portable.

Reply:
I agree we'll need to make changes to the mappings editor to make it reusable outside of the mappings editor. At this point, I'm concerned that we'll create confusion for maintainers if we have different ways of doing the same thing. I also would like the chance to share my thoughts around referencing the `documentationService` inside of the parameter components.  For the sake of keeping things consistent, would you be OK with sticking with the [existing pattern](https://github.com/elastic/kibana/blob/master/x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/field_parameters/copy_to_parameter.tsx#L29), just for now, until we've had a chance to talk about this as a group?

Reply:
I think it started to get confusing when we started added those methods in the index management `documentation` class. Since then we declare the field `type` documentation in one place, the constant file (`data_types_definition.ts`) but for some reason, the parameter documentation links are in a class (outside the mappings editor scope) so now we have 2 places to look at.

I am OK to follow the existing pattern and to discuss it as a group to put things back as they were but we did not discuss to split them in 2 places as there are now  😊